### DISCUSSION TITLE
Can I use Gemini or Claude instead of chatGPT 4

### DISCUSSION THREAD

**Discussion by haphanhp** [brianpetro/obsidian-smart-connections#536] [General]
because claude is free for PDF, can we have more than one API from different AI service?

**Comment by brianpetro** [comment]
Hi @haphanhp 

Currently, the only models available in the public stable release version are from OpenAI.

However, the early release, which is in beta testing with supporters, has more models available.

You can see the available models here: https://github.com/brianpetro/obsidian-smart-connections?tab=readme-ov-file#chat-models

🌴

**Reply by haphanhp** [reply]
before I can use ChatGPT 3.5 in December 2023 but now it not possible anymore? I changed new API key but still invalid (no smart connection found)
